.github/workflows/ci.yml

```diff
@@ -12,7 +12,33 @@ env:
   RUST_FEATURES: "rodio-backend,lyric-finder,media-control,image,notify,clipboard"
```

Comment:
I realize once you surround the arguments with quotes, spaces can be used instead of commas. See my below suggestions

Reply:
then you also need to update this variable to use spaces